tools/CreateMappings_rules.json

```diff
@@ -680,5 +680,13 @@
   {
     "alias": "Datadog",
     "regex": "Datadog"
+  },
+  {
+    "regex": "ChangeAnalysis",
+    "alias": "ChangeAnalysis"
+  },
+  {
+    "alias": "Purview",
+    "regex": "Purview"
```

Comment:
I think we agreed to use `module` instead of `regex`. Let me know your thoughts cc @VeryEarly @BethanyZhou